code/modules/food_and_drinks/machinery/coffeemaker.dm

```diff
@@ -16,13 +16,17 @@
 	/// The type path to instantiate for the coffee cartridge the device initially comes with, eg. /obj/item/coffee_cartridge
 	var/initial_cartridge = /obj/item/coffee_cartridge
 	/// The number of cups left
-	var/coffee_cups = 20
+	var/coffee_cups = 15
```

Comment:
Not a huge deal, but why not mention these in the PR?

Reply:
Uh, pardon me. I discussed the adjustments with the creator of  #68178 but I can mention this and all the other small tweaks in the description of the pr if that is needed

Reply:
Not a problem! If you discussed it with EOB that's fine. Was just curious.